korman/properties/modifiers/gui.py

```diff
@@ -633,6 +641,34 @@ def _create_moul_nodes(self, clickable_object, nodes, linkingnode, age_name, clk
         linking_panel_name.value = self.link_destination if self.link_destination else self.age_name
         linking_panel_name.link_output(linkingnode, "pfm", "TargetAge")
 
+        # Share MSB
+        if self.shareable:
+        # Region
+            share_region = nodes.new("PlasmaVolumeSensorNode")
+            if self.shr_region is None:
+                share_region.region_object = clk_region
+            else:
+                share_region.region_object = self.shr_region
+            share_region.bounds = "hull"
```

Comment:
Don't hardcode this - the physics exporter will grab the settings from the collision modifier, if it's enabled. We don't want to clobber any user settings :)